app/src/db/migrations/20240904140843_047-extend-submitter-role.js

```diff
@@ -0,0 +1,34 @@
+const uuid = require('uuid');
+
+const { Permissions, Roles } = require('../../forms/common/constants');
+
+const CREATED_BY = 'migration-047';
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function(knex) {
+    return Promise.resolve().then(() => {
+        const rolePermission = {
+          id: uuid.v4(),
+          createdBy: CREATED_BY,
+          role: Roles.FORM_SUBMITTER,
+          permission: Permissions.DOCUMENT_TEMPLATE_READ,
```

Comment:
@jasonchung1871 - This is going to make the `/rbac/current` call have a larger body (call is currently 840ms in prod). Any concerns about hitting a limit to how big it can be? Maybe we need to look at FORMS-1000 soon, to drop the load on the database?

Reply:
If we can figure out exactly what roles are needed and when, maybe we can retrieve the roles as we need them.